install.sh

```diff
@@ -236,6 +236,10 @@ do_install_rpm() {
             # We are operating with a version-based channel, so we should parse our version out
             rke2_majmin=$(echo "${INSTALL_RKE2_CHANNEL}" | sed -E -e "s/^v([0-9]+\.[0-9]+).*/\1/")
             rke2_rpm_channel=$(echo "${INSTALL_RKE2_CHANNEL}" | sed -E -e "s/^v[0-9]+\.[0-9]+-(.*)/\1/")
+            # If our regex fails to capture a "sane" channel out of the specified channel, fall back to `stable`
+            if [ "${rke2_rpm_channel}" = ${INSTALL_RKE2_CHANNEL} ]; then
```

Comment:
can we log that when falling back to stable so that the user understands what happened

Reply:
@galal-hussein I just pushed a commit that adds the message if we fall back to stable